Add clarification on managing team membership through SSO, and the SSO Team ID (#2308)
hashicorp/terraform-website: content/cloud-docs/users-teams-organizations/single-sign-on/azure-ad.mdx

1. Edit step 2, "User Attributes & Claims."
   We recommend naming it "MemberOf", leaving the namespace blank, and potentially sourcing `user.assignedroles` as an easy starting point.

> -> **Note:** When Azure AD is configured to use Group Claims it will provide Group UUIDs rather than human readable names in its SAML assertions. When managing team membersion through Azure AD we recommend [configuring SSO Team IDs](/cloud-docs/users-teams-organizations/single-sign-on#team-names-and-sso-team-ids) for your Terraform Cloud teams to match the Group UUIDs provided by Azure.

Reviewer: ```suggestion
-> **Note:** When Azure AD is configured to use Group Claims, it provides Group UUIDs instead of human readable names in its SAML assertions. We recommend [configuring SSO Team IDs](/cloud-docs/users-teams-organizations/single-sign-on#team-names-and-sso-team-ids) for your Terraform Cloud teams to match these Azure Group UUIDs.
```
